### ISSUE TITLE
Fix broken `linalg.norm` matrix order +/-2 and nuclear norm for extreme values

### ISSUE THREAD

**Issue by kurtamohler** [pytorch/pytorch#52633]
PR #48436 broke some of the extreme value behavior for `linalg.norm`, when`svd` is called internally: https://github.com/pytorch/pytorch/pull/48436#issuecomment-741149670

**Comment by cpuhrsch** [comment]
@kurtamohler - is this still an issue?

**Comment by kurtamohler** [comment]
@cpuhrsch, yep, all the cases mentioned in this comment are still failing: https://github.com/pytorch/pytorch/pull/48436#issuecomment-741149670

**Comment by cpuhrsch** [comment]
This seems like something we should review.

**Comment by mruberry** [comment]
@ngimel: let's update this test to verify we're returning something reasonable
@IvanYashchuk: this type of input on CPU may cause a runtime error (!) -- let's verify
@mruberry: this may be a documentation update
